soroban-env-host/src/auth.rs

```diff
@@ -821,6 +825,35 @@ impl AuthorizationManager {
                         ));
                     }
                 }
+                // If there is no active tracker for this exact address object,
+                // try to find any matching active tracker for the address.
+                for tracker in self.try_borrow_account_trackers(host)?.iter() {
+                    if let Ok(mut tracker) = tracker.try_borrow_mut() {
+                        if !host.compare(&tracker.address, &address)?.is_eq() {
```

Comment:
Is it possible to add a test for the scenario being fixed?

Reply:
Yes, but that's more involved and I don't want to block RC2 PR. I'm working on tests for main though.

Reply:
If the tests are soon, we'll have less churn if we make sure this works with them before starting the release process.